querier.hh

```diff
@@ -51,13 +54,23 @@ auto consume_page(flat_mutation_reader_v2& reader,
 class querier_base {
     friend class querier_utils;
 
+public:
+    struct querier_config {
+        uint32_t tombstone_warn_threshold; // 0 disabled
+        querier_config()
```

Comment:
nit: `querier_config() = default;` also works (and is shorter).

Reply:
ok, fixed.